{
  "checkpoints": [
    "{\n  \"stub\": false,\n  \"trainable_name\": \"DEFAULT\",\n  \"trial_id\": \"5aaa5_00002\",\n  \"config\": {\n    \"l1\": 224,\n    \"l2\": 20,\n    \"lr\": 0.0002050918538970317,\n    \"batch_size\": 256\n  },\n  \"local_dir\": \"/DATA/chowdari1/ray_results/DEFAULT_2022-02-09_01-34-42\",\n  \"evaluated_params\": {\n    \"l1\": 224,\n    \"l2\": 20,\n    \"lr\": 0.0002050918538970317,\n    \"batch_size\": 256\n  },\n  \"experiment_tag\": \"2_batch_size=256,l1=224,l2=20,lr=0.00020509\",\n  \"location\": {\n    \"_type\": \"CLOUDPICKLE_FALLBACK\",\n    \"value\": \"8005953a000000000000008c0e7261792e74756e652e747269616c948c084c6f636174696f6e9493942981947d94288c08686f73746e616d65944e8c03706964944e75622e\"\n  },\n  \"placement_group_factory\": \"800595b6000000000000008c1f7261792e74756e652e7574696c732e706c6163656d656e745f67726f757073948c15506c6163656d656e7447726f7570466163746f72799493942981947d94288c155f686561645f62756e646c655f69735f656d70747994898c085f62756e646c6573945d947d94288c03435055944740000000000000008c0347505594473fe000000000000075618c095f7374726174656779948c045041434b948c055f6172677394298c075f6b7761726773947d9475622e\",\n  \"stopping_criterion\": {},\n  \"log_to_file\": [\n    null,\n    null\n  ],\n  \"max_failures\": 0,\n  \"_last_result\": {\n    \"trial_id\": \"5aaa5_00002\"\n  },\n  \"_default_result_or_future\": null,\n  \"last_update_time\": -Infinity,\n  \"metric_analysis\": {},\n  \"n_steps\": [\n    5,\n    10\n  ],\n  \"metric_n_steps\": {},\n  \"export_formats\": [],\n  \"status\": \"ERROR\",\n  \"start_time\": 1644350685.0871828,\n  \"logdir\": \"/DATA/chowdari1/ray_results/DEFAULT_2022-02-09_01-34-42/DEFAULT_5aaa5_00002_2_batch_size=256,l1=224,l2=20,lr=0.00020509_2022-02-09_01-34-45\",\n  \"runner\": null,\n  \"last_debug\": 0,\n  \"error_file\": \"/DATA/chowdari1/ray_results/DEFAULT_2022-02-09_01-34-42/DEFAULT_5aaa5_00002_2_batch_size=256,l1=224,l2=20,lr=0.00020509_2022-02-09_01-34-45/error.txt\",\n  \"error_msg\": \"Traceback (most recent call last):\\n  File \\\"/DATA/chowdari1/.local/lib/python3.7/site-packages/ray/tune/trial_runner.py\\\", line 924, in _process_trial\\n    results = self.trial_executor.fetch_result(trial)\\n  File \\\"/DATA/chowdari1/.local/lib/python3.7/site-packages/ray/tune/ray_trial_executor.py\\\", line 787, in fetch_result\\n    result = ray.get(trial_future[0], timeout=DEFAULT_GET_TIMEOUT)\\n  File \\\"/DATA/chowdari1/.local/lib/python3.7/site-packages/ray/_private/client_mode_hook.py\\\", line 105, in wrapper\\n    return func(*args, **kwargs)\\n  File \\\"/DATA/chowdari1/.local/lib/python3.7/site-packages/ray/worker.py\\\", line 1713, in get\\n    raise value.as_instanceof_cause()\\nray.exceptions.RayTaskError(TuneError): \\u001b[36mray::ImplicitFunc.train_buffered()\\u001b[39m (pid=19773, ip=172.25.0.208, repr=<types.ImplicitFunc object at 0x7fbe4cf85a20>)\\n  File \\\"/DATA/chowdari1/.local/lib/python3.7/site-packages/ray/tune/trainable.py\\\", line 255, in train_buffered\\n    result = self.train()\\n  File \\\"/DATA/chowdari1/.local/lib/python3.7/site-packages/ray/tune/trainable.py\\\", line 314, in train\\n    result = self.step()\\n  File \\\"/DATA/chowdari1/.local/lib/python3.7/site-packages/ray/tune/function_runner.py\\\", line 381, in step\\n    self._report_thread_runner_error(block=True)\\n  File \\\"/DATA/chowdari1/.local/lib/python3.7/site-packages/ray/tune/function_runner.py\\\", line 532, in _report_thread_runner_error\\n    (\\\"Trial raised an exception. Traceback:\\\\n{}\\\".format(err_tb_str)\\nray.tune.error.TuneError: Trial raised an exception. Traceback:\\n\\u001b[36mray::ImplicitFunc.train_buffered()\\u001b[39m (pid=19773, ip=172.25.0.208, repr=<types.ImplicitFunc object at 0x7fbe4cf85a20>)\\n  File \\\"/DATA/chowdari1/.local/lib/python3.7/site-packages/ray/tune/function_runner.py\\\", line 262, in run\\n    self._entrypoint()\\n  File \\\"/DATA/chowdari1/.local/lib/python3.7/site-packages/ray/tune/function_runner.py\\\", line 331, in entrypoint\\n    self._status_reporter.get_checkpoint())\\n  File \\\"/DATA/chowdari1/.local/lib/python3.7/site-packages/ray/tune/function_runner.py\\\", line 597, in _trainable_func\\n    output = fn()\\n  File \\\"/tmp/ipykernel_3885/2932301558.py\\\", line 4, in train\\n  File \\\"/tmp/ipykernel_3885/1736512985.py\\\", line 6, in __init__\\nNameError: name 'VariationalEncoder' is not defined\\n\",\n  \"trial_name_creator\": null,\n  \"custom_trial_name\": null,\n  \"custom_dirname\": null,\n  \"saving_to\": null,\n  \"remote_checkpoint_dir_prefix\": null,\n  \"sync_function_tpl\": null,\n  \"checkpoint_freq\": 0,\n  \"checkpoint_at_end\": false,\n  \"keep_checkpoints_num\": null,\n  \"checkpoint_score_attr\": \"training_iteration\",\n  \"sync_on_checkpoint\": true,\n  \"checkpoint_manager\": {\n    \"_type\": \"CLOUDPICKLE_FALLBACK\",\n    \"value\": \"80059566010000000000008c1b7261792e74756e652e636865636b706f696e745f6d616e61676572948c11436865636b706f696e744d616e616765729493942981947d94288c146b6565705f636865636b706f696e74735f6e756d94477ff00000000000008c165f636865636b706f696e745f73636f72655f6465736394898c165f636865636b706f696e745f73636f72655f61747472948c12747261696e696e675f697465726174696f6e948c1c6e65776573745f70657273697374656e745f636865636b706f696e749468008c0a436865636b706f696e749493942981947d94288c0773746f72616765948c0a70657273697374656e74948c0576616c7565944e8c06726573756c74947d9475628c195f6e65776573745f6d656d6f72795f636865636b706f696e7494680b2981947d9428680e8c066d656d6f72799468104e68117d9475628c115f626573745f636865636b706f696e7473945d948c0b5f6d656d62657273686970948f9475622e\"\n  },\n  \"restore_path\": null,\n  \"restoring_from\": null,\n  \"num_failures\": 1,\n  \"has_new_resources\": false,\n  \"results\": \"80054e2e\",\n  \"best_result\": \"80054e2e\",\n  \"param_config\": \"80054e2e\",\n  \"extra_arg\": \"80054e2e\",\n  \"_state_json\": null,\n  \"_state_valid\": false\n}",
    "{\n  \"stub\": false,\n  \"trainable_name\": \"DEFAULT\",\n  \"trial_id\": \"5aaa5_00000\",\n  \"config\": {\n    \"l1\": 224,\n    \"l2\": 20,\n    \"lr\": 0.005289652342471245,\n    \"batch_size\": 256\n  },\n  \"local_dir\": \"/DATA/chowdari1/ray_results/DEFAULT_2022-02-09_01-34-42\",\n  \"evaluated_params\": {\n    \"l1\": 224,\n    \"l2\": 20,\n    \"lr\": 0.005289652342471245,\n    \"batch_size\": 256\n  },\n  \"experiment_tag\": \"0_batch_size=256,l1=224,l2=20,lr=0.0052897\",\n  \"location\": {\n    \"_type\": \"CLOUDPICKLE_FALLBACK\",\n    \"value\": \"8005953a000000000000008c0e7261792e74756e652e747269616c948c084c6f636174696f6e9493942981947d94288c08686f73746e616d65944e8c03706964944e75622e\"\n  },\n  \"placement_group_factory\": \"800595b6000000000000008c1f7261792e74756e652e7574696c732e706c6163656d656e745f67726f757073948c15506c6163656d656e7447726f7570466163746f72799493942981947d94288c155f686561645f62756e646c655f69735f656d70747994898c085f62756e646c6573945d947d94288c03435055944740000000000000008c0347505594473fe000000000000075618c095f7374726174656779948c045041434b948c055f6172677394298c075f6b7761726773947d9475622e\",\n  \"stopping_criterion\": {},\n  \"log_to_file\": [\n    null,\n    null\n  ],\n  \"max_failures\": 0,\n  \"_last_result\": {},\n  \"_default_result_or_future\": null,\n  \"last_update_time\": -Infinity,\n  \"metric_analysis\": {},\n  \"n_steps\": [\n    5,\n    10\n  ],\n  \"metric_n_steps\": {},\n  \"export_formats\": [],\n  \"status\": \"ERROR\",\n  \"start_time\": 1644350683.0622773,\n  \"logdir\": \"/DATA/chowdari1/ray_results/DEFAULT_2022-02-09_01-34-42/DEFAULT_5aaa5_00000_0_batch_size=256,l1=224,l2=20,lr=0.0052897_2022-02-09_01-34-42\",\n  \"runner\": null,\n  \"last_debug\": 0,\n  \"error_file\": \"/DATA/chowdari1/ray_results/DEFAULT_2022-02-09_01-34-42/DEFAULT_5aaa5_00000_0_batch_size=256,l1=224,l2=20,lr=0.0052897_2022-02-09_01-34-42/error.txt\",\n  \"error_msg\": \"Traceback (most recent call last):\\n  File \\\"/DATA/chowdari1/.local/lib/python3.7/site-packages/ray/tune/trial_runner.py\\\", line 924, in _process_trial\\n    results = self.trial_executor.fetch_result(trial)\\n  File \\\"/DATA/chowdari1/.local/lib/python3.7/site-packages/ray/tune/ray_trial_executor.py\\\", line 787, in fetch_result\\n    result = ray.get(trial_future[0], timeout=DEFAULT_GET_TIMEOUT)\\n  File \\\"/DATA/chowdari1/.local/lib/python3.7/site-packages/ray/_private/client_mode_hook.py\\\", line 105, in wrapper\\n    return func(*args, **kwargs)\\n  File \\\"/DATA/chowdari1/.local/lib/python3.7/site-packages/ray/worker.py\\\", line 1713, in get\\n    raise value.as_instanceof_cause()\\nray.exceptions.RayTaskError(TuneError): \\u001b[36mray::ImplicitFunc.train_buffered()\\u001b[39m (pid=19759, ip=172.25.0.208, repr=<types.ImplicitFunc object at 0x7f0fc5186b70>)\\n  File \\\"/DATA/chowdari1/.local/lib/python3.7/site-packages/ray/tune/trainable.py\\\", line 255, in train_buffered\\n    result = self.train()\\n  File \\\"/DATA/chowdari1/.local/lib/python3.7/site-packages/ray/tune/trainable.py\\\", line 314, in train\\n    result = self.step()\\n  File \\\"/DATA/chowdari1/.local/lib/python3.7/site-packages/ray/tune/function_runner.py\\\", line 381, in step\\n    self._report_thread_runner_error(block=True)\\n  File \\\"/DATA/chowdari1/.local/lib/python3.7/site-packages/ray/tune/function_runner.py\\\", line 532, in _report_thread_runner_error\\n    (\\\"Trial raised an exception. Traceback:\\\\n{}\\\".format(err_tb_str)\\nray.tune.error.TuneError: Trial raised an exception. Traceback:\\n\\u001b[36mray::ImplicitFunc.train_buffered()\\u001b[39m (pid=19759, ip=172.25.0.208, repr=<types.ImplicitFunc object at 0x7f0fc5186b70>)\\n  File \\\"/DATA/chowdari1/.local/lib/python3.7/site-packages/ray/tune/function_runner.py\\\", line 262, in run\\n    self._entrypoint()\\n  File \\\"/DATA/chowdari1/.local/lib/python3.7/site-packages/ray/tune/function_runner.py\\\", line 331, in entrypoint\\n    self._status_reporter.get_checkpoint())\\n  File \\\"/DATA/chowdari1/.local/lib/python3.7/site-packages/ray/tune/function_runner.py\\\", line 597, in _trainable_func\\n    output = fn()\\n  File \\\"/tmp/ipykernel_3885/2932301558.py\\\", line 4, in train\\n  File \\\"/tmp/ipykernel_3885/1736512985.py\\\", line 6, in __init__\\nNameError: name 'VariationalEncoder' is not defined\\n\",\n  \"trial_name_creator\": null,\n  \"custom_trial_name\": null,\n  \"custom_dirname\": null,\n  \"saving_to\": null,\n  \"remote_checkpoint_dir_prefix\": null,\n  \"sync_function_tpl\": null,\n  \"checkpoint_freq\": 0,\n  \"checkpoint_at_end\": false,\n  \"keep_checkpoints_num\": null,\n  \"checkpoint_score_attr\": \"training_iteration\",\n  \"sync_on_checkpoint\": true,\n  \"checkpoint_manager\": {\n    \"_type\": \"CLOUDPICKLE_FALLBACK\",\n    \"value\": \"80059566010000000000008c1b7261792e74756e652e636865636b706f696e745f6d616e61676572948c11436865636b706f696e744d616e616765729493942981947d94288c146b6565705f636865636b706f696e74735f6e756d94477ff00000000000008c165f636865636b706f696e745f73636f72655f6465736394898c165f636865636b706f696e745f73636f72655f61747472948c12747261696e696e675f697465726174696f6e948c1c6e65776573745f70657273697374656e745f636865636b706f696e749468008c0a436865636b706f696e749493942981947d94288c0773746f72616765948c0a70657273697374656e74948c0576616c7565944e8c06726573756c74947d9475628c195f6e65776573745f6d656d6f72795f636865636b706f696e7494680b2981947d9428680e8c066d656d6f72799468104e68117d9475628c115f626573745f636865636b706f696e7473945d948c0b5f6d656d62657273686970948f9475622e\"\n  },\n  \"restore_path\": null,\n  \"restoring_from\": null,\n  \"num_failures\": 1,\n  \"has_new_resources\": false,\n  \"results\": \"80054e2e\",\n  \"best_result\": \"80054e2e\",\n  \"param_config\": \"80054e2e\",\n  \"extra_arg\": \"80054e2e\",\n  \"_state_json\": null,\n  \"_state_valid\": false\n}",
    "{\n  \"stub\": false,\n  \"trainable_name\": \"DEFAULT\",\n  \"trial_id\": \"5aaa5_00004\",\n  \"config\": {\n    \"l1\": 224,\n    \"l2\": 4,\n    \"lr\": 0.0017367224822684032,\n    \"batch_size\": 256\n  },\n  \"local_dir\": \"/DATA/chowdari1/ray_results/DEFAULT_2022-02-09_01-34-42\",\n  \"evaluated_params\": {\n    \"l1\": 224,\n    \"l2\": 4,\n    \"lr\": 0.0017367224822684032,\n    \"batch_size\": 256\n  },\n  \"experiment_tag\": \"4_batch_size=256,l1=224,l2=4,lr=0.0017367\",\n  \"location\": {\n    \"_type\": \"CLOUDPICKLE_FALLBACK\",\n    \"value\": \"8005953a000000000000008c0e7261792e74756e652e747269616c948c084c6f636174696f6e9493942981947d94288c08686f73746e616d65944e8c03706964944e75622e\"\n  },\n  \"placement_group_factory\": \"800595b6000000000000008c1f7261792e74756e652e7574696c732e706c6163656d656e745f67726f757073948c15506c6163656d656e7447726f7570466163746f72799493942981947d94288c155f686561645f62756e646c655f69735f656d70747994898c085f62756e646c6573945d947d94288c03435055944740000000000000008c0347505594473fe000000000000075618c095f7374726174656779948c045041434b948c055f6172677394298c075f6b7761726773947d9475622e\",\n  \"stopping_criterion\": {},\n  \"log_to_file\": [\n    null,\n    null\n  ],\n  \"max_failures\": 0,\n  \"_last_result\": {\n    \"trial_id\": \"5aaa5_00004\"\n  },\n  \"_default_result_or_future\": null,\n  \"last_update_time\": -Infinity,\n  \"metric_analysis\": {},\n  \"n_steps\": [\n    5,\n    10\n  ],\n  \"metric_n_steps\": {},\n  \"export_formats\": [],\n  \"status\": \"ERROR\",\n  \"start_time\": 1644350685.152668,\n  \"logdir\": \"/DATA/chowdari1/ray_results/DEFAULT_2022-02-09_01-34-42/DEFAULT_5aaa5_00004_4_batch_size=256,l1=224,l2=4,lr=0.0017367_2022-02-09_01-34-45\",\n  \"runner\": null,\n  \"last_debug\": 0,\n  \"error_file\": \"/DATA/chowdari1/ray_results/DEFAULT_2022-02-09_01-34-42/DEFAULT_5aaa5_00004_4_batch_size=256,l1=224,l2=4,lr=0.0017367_2022-02-09_01-34-45/error.txt\",\n  \"error_msg\": \"Traceback (most recent call last):\\n  File \\\"/DATA/chowdari1/.local/lib/python3.7/site-packages/ray/tune/trial_runner.py\\\", line 924, in _process_trial\\n    results = self.trial_executor.fetch_result(trial)\\n  File \\\"/DATA/chowdari1/.local/lib/python3.7/site-packages/ray/tune/ray_trial_executor.py\\\", line 787, in fetch_result\\n    result = ray.get(trial_future[0], timeout=DEFAULT_GET_TIMEOUT)\\n  File \\\"/DATA/chowdari1/.local/lib/python3.7/site-packages/ray/_private/client_mode_hook.py\\\", line 105, in wrapper\\n    return func(*args, **kwargs)\\n  File \\\"/DATA/chowdari1/.local/lib/python3.7/site-packages/ray/worker.py\\\", line 1713, in get\\n    raise value.as_instanceof_cause()\\nray.exceptions.RayTaskError(TuneError): \\u001b[36mray::ImplicitFunc.train_buffered()\\u001b[39m (pid=19742, ip=172.25.0.208, repr=<types.ImplicitFunc object at 0x7f98c5da2b70>)\\n  File \\\"/DATA/chowdari1/.local/lib/python3.7/site-packages/ray/tune/trainable.py\\\", line 255, in train_buffered\\n    result = self.train()\\n  File \\\"/DATA/chowdari1/.local/lib/python3.7/site-packages/ray/tune/trainable.py\\\", line 314, in train\\n    result = self.step()\\n  File \\\"/DATA/chowdari1/.local/lib/python3.7/site-packages/ray/tune/function_runner.py\\\", line 381, in step\\n    self._report_thread_runner_error(block=True)\\n  File \\\"/DATA/chowdari1/.local/lib/python3.7/site-packages/ray/tune/function_runner.py\\\", line 532, in _report_thread_runner_error\\n    (\\\"Trial raised an exception. Traceback:\\\\n{}\\\".format(err_tb_str)\\nray.tune.error.TuneError: Trial raised an exception. Traceback:\\n\\u001b[36mray::ImplicitFunc.train_buffered()\\u001b[39m (pid=19742, ip=172.25.0.208, repr=<types.ImplicitFunc object at 0x7f98c5da2b70>)\\n  File \\\"/DATA/chowdari1/.local/lib/python3.7/site-packages/ray/tune/function_runner.py\\\", line 262, in run\\n    self._entrypoint()\\n  File \\\"/DATA/chowdari1/.local/lib/python3.7/site-packages/ray/tune/function_runner.py\\\", line 331, in entrypoint\\n    self._status_reporter.get_checkpoint())\\n  File \\\"/DATA/chowdari1/.local/lib/python3.7/site-packages/ray/tune/function_runner.py\\\", line 597, in _trainable_func\\n    output = fn()\\n  File \\\"/tmp/ipykernel_3885/2932301558.py\\\", line 4, in train\\n  File \\\"/tmp/ipykernel_3885/1736512985.py\\\", line 6, in __init__\\nNameError: name 'VariationalEncoder' is not defined\\n\",\n  \"trial_name_creator\": null,\n  \"custom_trial_name\": null,\n  \"custom_dirname\": null,\n  \"saving_to\": null,\n  \"remote_checkpoint_dir_prefix\": null,\n  \"sync_function_tpl\": null,\n  \"checkpoint_freq\": 0,\n  \"checkpoint_at_end\": false,\n  \"keep_checkpoints_num\": null,\n  \"checkpoint_score_attr\": \"training_iteration\",\n  \"sync_on_checkpoint\": true,\n  \"checkpoint_manager\": {\n    \"_type\": \"CLOUDPICKLE_FALLBACK\",\n    \"value\": \"80059566010000000000008c1b7261792e74756e652e636865636b706f696e745f6d616e61676572948c11436865636b706f696e744d616e616765729493942981947d94288c146b6565705f636865636b706f696e74735f6e756d94477ff00000000000008c165f636865636b706f696e745f73636f72655f6465736394898c165f636865636b706f696e745f73636f72655f61747472948c12747261696e696e675f697465726174696f6e948c1c6e65776573745f70657273697374656e745f636865636b706f696e749468008c0a436865636b706f696e749493942981947d94288c0773746f72616765948c0a70657273697374656e74948c0576616c7565944e8c06726573756c74947d9475628c195f6e65776573745f6d656d6f72795f636865636b706f696e7494680b2981947d9428680e8c066d656d6f72799468104e68117d9475628c115f626573745f636865636b706f696e7473945d948c0b5f6d656d62657273686970948f9475622e\"\n  },\n  \"restore_path\": null,\n  \"restoring_from\": null,\n  \"num_failures\": 1,\n  \"has_new_resources\": false,\n  \"results\": \"80054e2e\",\n  \"best_result\": \"80054e2e\",\n  \"param_config\": \"80054e2e\",\n  \"extra_arg\": \"80054e2e\",\n  \"_state_json\": null,\n  \"_state_valid\": false\n}",
    "{\n  \"stub\": false,\n  \"trainable_name\": \"DEFAULT\",\n  \"trial_id\": \"5aaa5_00003\",\n  \"config\": {\n    \"l1\": 224,\n    \"l2\": 20,\n    \"lr\": 0.0014933613519117588,\n    \"batch_size\": 256\n  },\n  \"local_dir\": \"/DATA/chowdari1/ray_results/DEFAULT_2022-02-09_01-34-42\",\n  \"evaluated_params\": {\n    \"l1\": 224,\n    \"l2\": 20,\n    \"lr\": 0.0014933613519117588,\n    \"batch_size\": 256\n  },\n  \"experiment_tag\": \"3_batch_size=256,l1=224,l2=20,lr=0.0014934\",\n  \"location\": {\n    \"_type\": \"CLOUDPICKLE_FALLBACK\",\n    \"value\": \"8005953a000000000000008c0e7261792e74756e652e747269616c948c084c6f636174696f6e9493942981947d94288c08686f73746e616d65944e8c03706964944e75622e\"\n  },\n  \"placement_group_factory\": \"800595b6000000000000008c1f7261792e74756e652e7574696c732e706c6163656d656e745f67726f757073948c15506c6163656d656e7447726f7570466163746f72799493942981947d94288c155f686561645f62756e646c655f69735f656d70747994898c085f62756e646c6573945d947d94288c03435055944740000000000000008c0347505594473fe000000000000075618c095f7374726174656779948c045041434b948c055f6172677394298c075f6b7761726773947d9475622e\",\n  \"stopping_criterion\": {},\n  \"log_to_file\": [\n    null,\n    null\n  ],\n  \"max_failures\": 0,\n  \"_last_result\": {\n    \"trial_id\": \"5aaa5_00003\"\n  },\n  \"_default_result_or_future\": null,\n  \"last_update_time\": -Infinity,\n  \"metric_analysis\": {},\n  \"n_steps\": [\n    5,\n    10\n  ],\n  \"metric_n_steps\": {},\n  \"export_formats\": [],\n  \"status\": \"ERROR\",\n  \"start_time\": 1644350685.1159074,\n  \"logdir\": \"/DATA/chowdari1/ray_results/DEFAULT_2022-02-09_01-34-42/DEFAULT_5aaa5_00003_3_batch_size=256,l1=224,l2=20,lr=0.0014934_2022-02-09_01-34-45\",\n  \"runner\": null,\n  \"last_debug\": 0,\n  \"error_file\": \"/DATA/chowdari1/ray_results/DEFAULT_2022-02-09_01-34-42/DEFAULT_5aaa5_00003_3_batch_size=256,l1=224,l2=20,lr=0.0014934_2022-02-09_01-34-45/error.txt\",\n  \"error_msg\": \"Traceback (most recent call last):\\n  File \\\"/DATA/chowdari1/.local/lib/python3.7/site-packages/ray/tune/trial_runner.py\\\", line 924, in _process_trial\\n    results = self.trial_executor.fetch_result(trial)\\n  File \\\"/DATA/chowdari1/.local/lib/python3.7/site-packages/ray/tune/ray_trial_executor.py\\\", line 787, in fetch_result\\n    result = ray.get(trial_future[0], timeout=DEFAULT_GET_TIMEOUT)\\n  File \\\"/DATA/chowdari1/.local/lib/python3.7/site-packages/ray/_private/client_mode_hook.py\\\", line 105, in wrapper\\n    return func(*args, **kwargs)\\n  File \\\"/DATA/chowdari1/.local/lib/python3.7/site-packages/ray/worker.py\\\", line 1713, in get\\n    raise value.as_instanceof_cause()\\nray.exceptions.RayTaskError(TuneError): \\u001b[36mray::ImplicitFunc.train_buffered()\\u001b[39m (pid=19649, ip=172.25.0.208, repr=<types.ImplicitFunc object at 0x7fdb6c5c7b00>)\\n  File \\\"/DATA/chowdari1/.local/lib/python3.7/site-packages/ray/tune/trainable.py\\\", line 255, in train_buffered\\n    result = self.train()\\n  File \\\"/DATA/chowdari1/.local/lib/python3.7/site-packages/ray/tune/trainable.py\\\", line 314, in train\\n    result = self.step()\\n  File \\\"/DATA/chowdari1/.local/lib/python3.7/site-packages/ray/tune/function_runner.py\\\", line 381, in step\\n    self._report_thread_runner_error(block=True)\\n  File \\\"/DATA/chowdari1/.local/lib/python3.7/site-packages/ray/tune/function_runner.py\\\", line 532, in _report_thread_runner_error\\n    (\\\"Trial raised an exception. Traceback:\\\\n{}\\\".format(err_tb_str)\\nray.tune.error.TuneError: Trial raised an exception. Traceback:\\n\\u001b[36mray::ImplicitFunc.train_buffered()\\u001b[39m (pid=19649, ip=172.25.0.208, repr=<types.ImplicitFunc object at 0x7fdb6c5c7b00>)\\n  File \\\"/DATA/chowdari1/.local/lib/python3.7/site-packages/ray/tune/function_runner.py\\\", line 262, in run\\n    self._entrypoint()\\n  File \\\"/DATA/chowdari1/.local/lib/python3.7/site-packages/ray/tune/function_runner.py\\\", line 331, in entrypoint\\n    self._status_reporter.get_checkpoint())\\n  File \\\"/DATA/chowdari1/.local/lib/python3.7/site-packages/ray/tune/function_runner.py\\\", line 597, in _trainable_func\\n    output = fn()\\n  File \\\"/tmp/ipykernel_3885/2932301558.py\\\", line 4, in train\\n  File \\\"/tmp/ipykernel_3885/1736512985.py\\\", line 6, in __init__\\nNameError: name 'VariationalEncoder' is not defined\\n\",\n  \"trial_name_creator\": null,\n  \"custom_trial_name\": null,\n  \"custom_dirname\": null,\n  \"saving_to\": null,\n  \"remote_checkpoint_dir_prefix\": null,\n  \"sync_function_tpl\": null,\n  \"checkpoint_freq\": 0,\n  \"checkpoint_at_end\": false,\n  \"keep_checkpoints_num\": null,\n  \"checkpoint_score_attr\": \"training_iteration\",\n  \"sync_on_checkpoint\": true,\n  \"checkpoint_manager\": {\n    \"_type\": \"CLOUDPICKLE_FALLBACK\",\n    \"value\": \"80059566010000000000008c1b7261792e74756e652e636865636b706f696e745f6d616e61676572948c11436865636b706f696e744d616e616765729493942981947d94288c146b6565705f636865636b706f696e74735f6e756d94477ff00000000000008c165f636865636b706f696e745f73636f72655f6465736394898c165f636865636b706f696e745f73636f72655f61747472948c12747261696e696e675f697465726174696f6e948c1c6e65776573745f70657273697374656e745f636865636b706f696e749468008c0a436865636b706f696e749493942981947d94288c0773746f72616765948c0a70657273697374656e74948c0576616c7565944e8c06726573756c74947d9475628c195f6e65776573745f6d656d6f72795f636865636b706f696e7494680b2981947d9428680e8c066d656d6f72799468104e68117d9475628c115f626573745f636865636b706f696e7473945d948c0b5f6d656d62657273686970948f9475622e\"\n  },\n  \"restore_path\": null,\n  \"restoring_from\": null,\n  \"num_failures\": 1,\n  \"has_new_resources\": false,\n  \"results\": \"80054e2e\",\n  \"best_result\": \"80054e2e\",\n  \"param_config\": \"80054e2e\",\n  \"extra_arg\": \"80054e2e\",\n  \"_state_json\": null,\n  \"_state_valid\": false\n}",
    "{\n  \"stub\": false,\n  \"trainable_name\": \"DEFAULT\",\n  \"trial_id\": \"5aaa5_00008\",\n  \"config\": {\n    \"l1\": 224,\n    \"l2\": 4,\n    \"lr\": 0.017391213169516362,\n    \"batch_size\": 256\n  },\n  \"local_dir\": \"/DATA/chowdari1/ray_results/DEFAULT_2022-02-09_01-34-42\",\n  \"evaluated_params\": {\n    \"l1\": 224,\n    \"l2\": 4,\n    \"lr\": 0.017391213169516362,\n    \"batch_size\": 256\n  },\n  \"experiment_tag\": \"8_batch_size=256,l1=224,l2=4,lr=0.017391\",\n  \"location\": {\n    \"_type\": \"CLOUDPICKLE_FALLBACK\",\n    \"value\": \"8005953a000000000000008c0e7261792e74756e652e747269616c948c084c6f636174696f6e9493942981947d94288c08686f73746e616d65944e8c03706964944e75622e\"\n  },\n  \"placement_group_factory\": \"800595b6000000000000008c1f7261792e74756e652e7574696c732e706c6163656d656e745f67726f757073948c15506c6163656d656e7447726f7570466163746f72799493942981947d94288c155f686561645f62756e646c655f69735f656d70747994898c085f62756e646c6573945d947d94288c03435055944740000000000000008c0347505594473fe000000000000075618c095f7374726174656779948c045041434b948c055f6172677394298c075f6b7761726773947d9475622e\",\n  \"stopping_criterion\": {},\n  \"log_to_file\": [\n    null,\n    null\n  ],\n  \"max_failures\": 0,\n  \"_last_result\": {\n    \"trial_id\": \"5aaa5_00008\"\n  },\n  \"_default_result_or_future\": null,\n  \"last_update_time\": -Infinity,\n  \"metric_analysis\": {},\n  \"n_steps\": [\n    5,\n    10\n  ],\n  \"metric_n_steps\": {},\n  \"export_formats\": [],\n  \"status\": \"ERROR\",\n  \"start_time\": 1644350685.310084,\n  \"logdir\": \"/DATA/chowdari1/ray_results/DEFAULT_2022-02-09_01-34-42/DEFAULT_5aaa5_00008_8_batch_size=256,l1=224,l2=4,lr=0.017391_2022-02-09_01-34-45\",\n  \"runner\": null,\n  \"last_debug\": 0,\n  \"error_file\": \"/DATA/chowdari1/ray_results/DEFAULT_2022-02-09_01-34-42/DEFAULT_5aaa5_00008_8_batch_size=256,l1=224,l2=4,lr=0.017391_2022-02-09_01-34-45/error.txt\",\n  \"error_msg\": \"Traceback (most recent call last):\\n  File \\\"/DATA/chowdari1/.local/lib/python3.7/site-packages/ray/tune/trial_runner.py\\\", line 924, in _process_trial\\n    results = self.trial_executor.fetch_result(trial)\\n  File \\\"/DATA/chowdari1/.local/lib/python3.7/site-packages/ray/tune/ray_trial_executor.py\\\", line 787, in fetch_result\\n    result = ray.get(trial_future[0], timeout=DEFAULT_GET_TIMEOUT)\\n  File \\\"/DATA/chowdari1/.local/lib/python3.7/site-packages/ray/_private/client_mode_hook.py\\\", line 105, in wrapper\\n    return func(*args, **kwargs)\\n  File \\\"/DATA/chowdari1/.local/lib/python3.7/site-packages/ray/worker.py\\\", line 1713, in get\\n    raise value.as_instanceof_cause()\\nray.exceptions.RayTaskError(TuneError): \\u001b[36mray::ImplicitFunc.train_buffered()\\u001b[39m (pid=19703, ip=172.25.0.208, repr=<types.ImplicitFunc object at 0x7fe38ad6ef28>)\\n  File \\\"/DATA/chowdari1/.local/lib/python3.7/site-packages/ray/tune/trainable.py\\\", line 255, in train_buffered\\n    result = self.train()\\n  File \\\"/DATA/chowdari1/.local/lib/python3.7/site-packages/ray/tune/trainable.py\\\", line 314, in train\\n    result = self.step()\\n  File \\\"/DATA/chowdari1/.local/lib/python3.7/site-packages/ray/tune/function_runner.py\\\", line 381, in step\\n    self._report_thread_runner_error(block=True)\\n  File \\\"/DATA/chowdari1/.local/lib/python3.7/site-packages/ray/tune/function_runner.py\\\", line 532, in _report_thread_runner_error\\n    (\\\"Trial raised an exception. Traceback:\\\\n{}\\\".format(err_tb_str)\\nray.tune.error.TuneError: Trial raised an exception. Traceback:\\n\\u001b[36mray::ImplicitFunc.train_buffered()\\u001b[39m (pid=19703, ip=172.25.0.208, repr=<types.ImplicitFunc object at 0x7fe38ad6ef28>)\\n  File \\\"/DATA/chowdari1/.local/lib/python3.7/site-packages/ray/tune/function_runner.py\\\", line 262, in run\\n    self._entrypoint()\\n  File \\\"/DATA/chowdari1/.local/lib/python3.7/site-packages/ray/tune/function_runner.py\\\", line 331, in entrypoint\\n    self._status_reporter.get_checkpoint())\\n  File \\\"/DATA/chowdari1/.local/lib/python3.7/site-packages/ray/tune/function_runner.py\\\", line 597, in _trainable_func\\n    output = fn()\\n  File \\\"/tmp/ipykernel_3885/2932301558.py\\\", line 4, in train\\n  File \\\"/tmp/ipykernel_3885/1736512985.py\\\", line 6, in __init__\\nNameError: name 'VariationalEncoder' is not defined\\n\",\n  \"trial_name_creator\": null,\n  \"custom_trial_name\": null,\n  \"custom_dirname\": null,\n  \"saving_to\": null,\n  \"remote_checkpoint_dir_prefix\": null,\n  \"sync_function_tpl\": null,\n  \"checkpoint_freq\": 0,\n  \"checkpoint_at_end\": false,\n  \"keep_checkpoints_num\": null,\n  \"checkpoint_score_attr\": \"training_iteration\",\n  \"sync_on_checkpoint\": true,\n  \"checkpoint_manager\": {\n    \"_type\": \"CLOUDPICKLE_FALLBACK\",\n    \"value\": \"80059566010000000000008c1b7261792e74756e652e636865636b706f696e745f6d616e61676572948c11436865636b706f696e744d616e616765729493942981947d94288c146b6565705f636865636b706f696e74735f6e756d94477ff00000000000008c165f636865636b706f696e745f73636f72655f6465736394898c165f636865636b706f696e745f73636f72655f61747472948c12747261696e696e675f697465726174696f6e948c1c6e65776573745f70657273697374656e745f636865636b706f696e749468008c0a436865636b706f696e749493942981947d94288c0773746f72616765948c0a70657273697374656e74948c0576616c7565944e8c06726573756c74947d9475628c195f6e65776573745f6d656d6f72795f636865636b706f696e7494680b2981947d9428680e8c066d656d6f72799468104e68117d9475628c115f626573745f636865636b706f696e7473945d948c0b5f6d656d62657273686970948f9475622e\"\n  },\n  \"restore_path\": null,\n  \"restoring_from\": null,\n  \"num_failures\": 1,\n  \"has_new_resources\": false,\n  \"results\": \"80054e2e\",\n  \"best_result\": \"80054e2e\",\n  \"param_config\": \"80054e2e\",\n  \"extra_arg\": \"80054e2e\",\n  \"_state_json\": null,\n  \"_state_valid\": false\n}",
    "{\n  \"stub\": false,\n  \"trainable_name\": \"DEFAULT\",\n  \"trial_id\": \"5aaa5_00007\",\n  \"config\": {\n    \"l1\": 224,\n    \"l2\": 20,\n    \"lr\": 0.005071592483026825,\n    \"batch_size\": 256\n  },\n  \"local_dir\": \"/DATA/chowdari1/ray_results/DEFAULT_2022-02-09_01-34-42\",\n  \"evaluated_params\": {\n    \"l1\": 224,\n    \"l2\": 20,\n    \"lr\": 0.005071592483026825,\n    \"batch_size\": 256\n  },\n  \"experiment_tag\": \"7_batch_size=256,l1=224,l2=20,lr=0.0050716\",\n  \"location\": {\n    \"_type\": \"CLOUDPICKLE_FALLBACK\",\n    \"value\": \"8005953a000000000000008c0e7261792e74756e652e747269616c948c084c6f636174696f6e9493942981947d94288c08686f73746e616d65944e8c03706964944e75622e\"\n  },\n  \"placement_group_factory\": \"800595b6000000000000008c1f7261792e74756e652e7574696c732e706c6163656d656e745f67726f757073948c15506c6163656d656e7447726f7570466163746f72799493942981947d94288c155f686561645f62756e646c655f69735f656d70747994898c085f62756e646c6573945d947d94288c03435055944740000000000000008c0347505594473fe000000000000075618c095f7374726174656779948c045041434b948c055f6172677394298c075f6b7761726773947d9475622e\",\n  \"stopping_criterion\": {},\n  \"log_to_file\": [\n    null,\n    null\n  ],\n  \"max_failures\": 0,\n  \"_last_result\": {\n    \"trial_id\": \"5aaa5_00007\"\n  },\n  \"_default_result_or_future\": null,\n  \"last_update_time\": -Infinity,\n  \"metric_analysis\": {},\n  \"n_steps\": [\n    5,\n    10\n  ],\n  \"metric_n_steps\": {},\n  \"export_formats\": [],\n  \"status\": \"ERROR\",\n  \"start_time\": 1644350685.2592194,\n  \"logdir\": \"/DATA/chowdari1/ray_results/DEFAULT_2022-02-09_01-34-42/DEFAULT_5aaa5_00007_7_batch_size=256,l1=224,l2=20,lr=0.0050716_2022-02-09_01-34-45\",\n  \"runner\": null,\n  \"last_debug\": 0,\n  \"error_file\": \"/DATA/chowdari1/ray_results/DEFAULT_2022-02-09_01-34-42/DEFAULT_5aaa5_00007_7_batch_size=256,l1=224,l2=20,lr=0.0050716_2022-02-09_01-34-45/error.txt\",\n  \"error_msg\": \"Traceback (most recent call last):\\n  File \\\"/DATA/chowdari1/.local/lib/python3.7/site-packages/ray/tune/trial_runner.py\\\", line 924, in _process_trial\\n    results = self.trial_executor.fetch_result(trial)\\n  File \\\"/DATA/chowdari1/.local/lib/python3.7/site-packages/ray/tune/ray_trial_executor.py\\\", line 787, in fetch_result\\n    result = ray.get(trial_future[0], timeout=DEFAULT_GET_TIMEOUT)\\n  File \\\"/DATA/chowdari1/.local/lib/python3.7/site-packages/ray/_private/client_mode_hook.py\\\", line 105, in wrapper\\n    return func(*args, **kwargs)\\n  File \\\"/DATA/chowdari1/.local/lib/python3.7/site-packages/ray/worker.py\\\", line 1713, in get\\n    raise value.as_instanceof_cause()\\nray.exceptions.RayTaskError(TuneError): \\u001b[36mray::ImplicitFunc.train_buffered()\\u001b[39m (pid=19713, ip=172.25.0.208, repr=<types.ImplicitFunc object at 0x7f86f4f77ac8>)\\n  File \\\"/DATA/chowdari1/.local/lib/python3.7/site-packages/ray/tune/trainable.py\\\", line 255, in train_buffered\\n    result = self.train()\\n  File \\\"/DATA/chowdari1/.local/lib/python3.7/site-packages/ray/tune/trainable.py\\\", line 314, in train\\n    result = self.step()\\n  File \\\"/DATA/chowdari1/.local/lib/python3.7/site-packages/ray/tune/function_runner.py\\\", line 381, in step\\n    self._report_thread_runner_error(block=True)\\n  File \\\"/DATA/chowdari1/.local/lib/python3.7/site-packages/ray/tune/function_runner.py\\\", line 532, in _report_thread_runner_error\\n    (\\\"Trial raised an exception. Traceback:\\\\n{}\\\".format(err_tb_str)\\nray.tune.error.TuneError: Trial raised an exception. Traceback:\\n\\u001b[36mray::ImplicitFunc.train_buffered()\\u001b[39m (pid=19713, ip=172.25.0.208, repr=<types.ImplicitFunc object at 0x7f86f4f77ac8>)\\n  File \\\"/DATA/chowdari1/.local/lib/python3.7/site-packages/ray/tune/function_runner.py\\\", line 262, in run\\n    self._entrypoint()\\n  File \\\"/DATA/chowdari1/.local/lib/python3.7/site-packages/ray/tune/function_runner.py\\\", line 331, in entrypoint\\n    self._status_reporter.get_checkpoint())\\n  File \\\"/DATA/chowdari1/.local/lib/python3.7/site-packages/ray/tune/function_runner.py\\\", line 597, in _trainable_func\\n    output = fn()\\n  File \\\"/tmp/ipykernel_3885/2932301558.py\\\", line 4, in train\\n  File \\\"/tmp/ipykernel_3885/1736512985.py\\\", line 6, in __init__\\nNameError: name 'VariationalEncoder' is not defined\\n\",\n  \"trial_name_creator\": null,\n  \"custom_trial_name\": null,\n  \"custom_dirname\": null,\n  \"saving_to\": null,\n  \"remote_checkpoint_dir_prefix\": null,\n  \"sync_function_tpl\": null,\n  \"checkpoint_freq\": 0,\n  \"checkpoint_at_end\": false,\n  \"keep_checkpoints_num\": null,\n  \"checkpoint_score_attr\": \"training_iteration\",\n  \"sync_on_checkpoint\": true,\n  \"checkpoint_manager\": {\n    \"_type\": \"CLOUDPICKLE_FALLBACK\",\n    \"value\": \"80059566010000000000008c1b7261792e74756e652e636865636b706f696e745f6d616e61676572948c11436865636b706f696e744d616e616765729493942981947d94288c146b6565705f636865636b706f696e74735f6e756d94477ff00000000000008c165f636865636b706f696e745f73636f72655f6465736394898c165f636865636b706f696e745f73636f72655f61747472948c12747261696e696e675f697465726174696f6e948c1c6e65776573745f70657273697374656e745f636865636b706f696e749468008c0a436865636b706f696e749493942981947d94288c0773746f72616765948c0a70657273697374656e74948c0576616c7565944e8c06726573756c74947d9475628c195f6e65776573745f6d656d6f72795f636865636b706f696e7494680b2981947d9428680e8c066d656d6f72799468104e68117d9475628c115f626573745f636865636b706f696e7473945d948c0b5f6d656d62657273686970948f9475622e\"\n  },\n  \"restore_path\": null,\n  \"restoring_from\": null,\n  \"num_failures\": 1,\n  \"has_new_resources\": false,\n  \"results\": \"80054e2e\",\n  \"best_result\": \"80054e2e\",\n  \"param_config\": \"80054e2e\",\n  \"extra_arg\": \"80054e2e\",\n  \"_state_json\": null,\n  \"_state_valid\": false\n}",
    "{\n  \"stub\": false,\n  \"trainable_name\": \"DEFAULT\",\n  \"trial_id\": \"5aaa5_00009\",\n  \"config\": {\n    \"l1\": 224,\n    \"l2\": 4,\n    \"lr\": 0.0017845739689726255,\n    \"batch_size\": 256\n  },\n  \"local_dir\": \"/DATA/chowdari1/ray_results/DEFAULT_2022-02-09_01-34-42\",\n  \"evaluated_params\": {\n    \"l1\": 224,\n    \"l2\": 4,\n    \"lr\": 0.0017845739689726255,\n    \"batch_size\": 256\n  },\n  \"experiment_tag\": \"9_batch_size=256,l1=224,l2=4,lr=0.0017846\",\n  \"location\": {\n    \"_type\": \"CLOUDPICKLE_FALLBACK\",\n    \"value\": \"8005953a000000000000008c0e7261792e74756e652e747269616c948c084c6f636174696f6e9493942981947d94288c08686f73746e616d65944e8c03706964944e75622e\"\n  },\n  \"placement_group_factory\": \"800595b6000000000000008c1f7261792e74756e652e7574696c732e706c6163656d656e745f67726f757073948c15506c6163656d656e7447726f7570466163746f72799493942981947d94288c155f686561645f62756e646c655f69735f656d70747994898c085f62756e646c6573945d947d94288c03435055944740000000000000008c0347505594473fe000000000000075618c095f7374726174656779948c045041434b948c055f6172677394298c075f6b7761726773947d9475622e\",\n  \"stopping_criterion\": {},\n  \"log_to_file\": [\n    null,\n    null\n  ],\n  \"max_failures\": 0,\n  \"_last_result\": {\n    \"trial_id\": \"5aaa5_00009\"\n  },\n  \"_default_result_or_future\": null,\n  \"last_update_time\": -Infinity,\n  \"metric_analysis\": {},\n  \"n_steps\": [\n    5,\n    10\n  ],\n  \"metric_n_steps\": {},\n  \"export_formats\": [],\n  \"status\": \"ERROR\",\n  \"start_time\": 1644350685.3532853,\n  \"logdir\": \"/DATA/chowdari1/ray_results/DEFAULT_2022-02-09_01-34-42/DEFAULT_5aaa5_00009_9_batch_size=256,l1=224,l2=4,lr=0.0017846_2022-02-09_01-34-45\",\n  \"runner\": null,\n  \"last_debug\": 0,\n  \"error_file\": \"/DATA/chowdari1/ray_results/DEFAULT_2022-02-09_01-34-42/DEFAULT_5aaa5_00009_9_batch_size=256,l1=224,l2=4,lr=0.0017846_2022-02-09_01-34-45/error.txt\",\n  \"error_msg\": \"Traceback (most recent call last):\\n  File \\\"/DATA/chowdari1/.local/lib/python3.7/site-packages/ray/tune/trial_runner.py\\\", line 924, in _process_trial\\n    results = self.trial_executor.fetch_result(trial)\\n  File \\\"/DATA/chowdari1/.local/lib/python3.7/site-packages/ray/tune/ray_trial_executor.py\\\", line 787, in fetch_result\\n    result = ray.get(trial_future[0], timeout=DEFAULT_GET_TIMEOUT)\\n  File \\\"/DATA/chowdari1/.local/lib/python3.7/site-packages/ray/_private/client_mode_hook.py\\\", line 105, in wrapper\\n    return func(*args, **kwargs)\\n  File \\\"/DATA/chowdari1/.local/lib/python3.7/site-packages/ray/worker.py\\\", line 1713, in get\\n    raise value.as_instanceof_cause()\\nray.exceptions.RayTaskError(TuneError): \\u001b[36mray::ImplicitFunc.train_buffered()\\u001b[39m (pid=19544, ip=172.25.0.208, repr=<types.ImplicitFunc object at 0x7f2feb5bca58>)\\n  File \\\"/DATA/chowdari1/.local/lib/python3.7/site-packages/ray/tune/trainable.py\\\", line 255, in train_buffered\\n    result = self.train()\\n  File \\\"/DATA/chowdari1/.local/lib/python3.7/site-packages/ray/tune/trainable.py\\\", line 314, in train\\n    result = self.step()\\n  File \\\"/DATA/chowdari1/.local/lib/python3.7/site-packages/ray/tune/function_runner.py\\\", line 381, in step\\n    self._report_thread_runner_error(block=True)\\n  File \\\"/DATA/chowdari1/.local/lib/python3.7/site-packages/ray/tune/function_runner.py\\\", line 532, in _report_thread_runner_error\\n    (\\\"Trial raised an exception. Traceback:\\\\n{}\\\".format(err_tb_str)\\nray.tune.error.TuneError: Trial raised an exception. Traceback:\\n\\u001b[36mray::ImplicitFunc.train_buffered()\\u001b[39m (pid=19544, ip=172.25.0.208, repr=<types.ImplicitFunc object at 0x7f2feb5bca58>)\\n  File \\\"/DATA/chowdari1/.local/lib/python3.7/site-packages/ray/tune/function_runner.py\\\", line 262, in run\\n    self._entrypoint()\\n  File \\\"/DATA/chowdari1/.local/lib/python3.7/site-packages/ray/tune/function_runner.py\\\", line 331, in entrypoint\\n    self._status_reporter.get_checkpoint())\\n  File \\\"/DATA/chowdari1/.local/lib/python3.7/site-packages/ray/tune/function_runner.py\\\", line 597, in _trainable_func\\n    output = fn()\\n  File \\\"/tmp/ipykernel_3885/2932301558.py\\\", line 4, in train\\n  File \\\"/tmp/ipykernel_3885/1736512985.py\\\", line 6, in __init__\\nNameError: name 'VariationalEncoder' is not defined\\n\",\n  \"trial_name_creator\": null,\n  \"custom_trial_name\": null,\n  \"custom_dirname\": null,\n  \"saving_to\": null,\n  \"remote_checkpoint_dir_prefix\": null,\n  \"sync_function_tpl\": null,\n  \"checkpoint_freq\": 0,\n  \"checkpoint_at_end\": false,\n  \"keep_checkpoints_num\": null,\n  \"checkpoint_score_attr\": \"training_iteration\",\n  \"sync_on_checkpoint\": true,\n  \"checkpoint_manager\": {\n    \"_type\": \"CLOUDPICKLE_FALLBACK\",\n    \"value\": \"80059566010000000000008c1b7261792e74756e652e636865636b706f696e745f6d616e61676572948c11436865636b706f696e744d616e616765729493942981947d94288c146b6565705f636865636b706f696e74735f6e756d94477ff00000000000008c165f636865636b706f696e745f73636f72655f6465736394898c165f636865636b706f696e745f73636f72655f61747472948c12747261696e696e675f697465726174696f6e948c1c6e65776573745f70657273697374656e745f636865636b706f696e749468008c0a436865636b706f696e749493942981947d94288c0773746f72616765948c0a70657273697374656e74948c0576616c7565944e8c06726573756c74947d9475628c195f6e65776573745f6d656d6f72795f636865636b706f696e7494680b2981947d9428680e8c066d656d6f72799468104e68117d9475628c115f626573745f636865636b706f696e7473945d948c0b5f6d656d62657273686970948f9475622e\"\n  },\n  \"restore_path\": null,\n  \"restoring_from\": null,\n  \"num_failures\": 1,\n  \"has_new_resources\": false,\n  \"results\": \"80054e2e\",\n  \"best_result\": \"80054e2e\",\n  \"param_config\": \"80054e2e\",\n  \"extra_arg\": \"80054e2e\",\n  \"_state_json\": null,\n  \"_state_valid\": false\n}",
    "{\n  \"stub\": false,\n  \"trainable_name\": \"DEFAULT\",\n  \"trial_id\": \"5aaa5_00006\",\n  \"config\": {\n    \"l1\": 224,\n    \"l2\": 4,\n    \"lr\": 0.03195114730201637,\n    \"batch_size\": 256\n  },\n  \"local_dir\": \"/DATA/chowdari1/ray_results/DEFAULT_2022-02-09_01-34-42\",\n  \"evaluated_params\": {\n    \"l1\": 224,\n    \"l2\": 4,\n    \"lr\": 0.03195114730201637,\n    \"batch_size\": 256\n  },\n  \"experiment_tag\": \"6_batch_size=256,l1=224,l2=4,lr=0.031951\",\n  \"location\": {\n    \"_type\": \"CLOUDPICKLE_FALLBACK\",\n    \"value\": \"8005953a000000000000008c0e7261792e74756e652e747269616c948c084c6f636174696f6e9493942981947d94288c08686f73746e616d65944e8c03706964944e75622e\"\n  },\n  \"placement_group_factory\": \"800595b6000000000000008c1f7261792e74756e652e7574696c732e706c6163656d656e745f67726f757073948c15506c6163656d656e7447726f7570466163746f72799493942981947d94288c155f686561645f62756e646c655f69735f656d70747994898c085f62756e646c6573945d947d94288c03435055944740000000000000008c0347505594473fe000000000000075618c095f7374726174656779948c045041434b948c055f6172677394298c075f6b7761726773947d9475622e\",\n  \"stopping_criterion\": {},\n  \"log_to_file\": [\n    null,\n    null\n  ],\n  \"max_failures\": 0,\n  \"_last_result\": {\n    \"trial_id\": \"5aaa5_00006\"\n  },\n  \"_default_result_or_future\": null,\n  \"last_update_time\": -Infinity,\n  \"metric_analysis\": {},\n  \"n_steps\": [\n    5,\n    10\n  ],\n  \"metric_n_steps\": {},\n  \"export_formats\": [],\n  \"status\": \"ERROR\",\n  \"start_time\": 1644350685.2223716,\n  \"logdir\": \"/DATA/chowdari1/ray_results/DEFAULT_2022-02-09_01-34-42/DEFAULT_5aaa5_00006_6_batch_size=256,l1=224,l2=4,lr=0.031951_2022-02-09_01-34-45\",\n  \"runner\": null,\n  \"last_debug\": 0,\n  \"error_file\": \"/DATA/chowdari1/ray_results/DEFAULT_2022-02-09_01-34-42/DEFAULT_5aaa5_00006_6_batch_size=256,l1=224,l2=4,lr=0.031951_2022-02-09_01-34-45/error.txt\",\n  \"error_msg\": \"Traceback (most recent call last):\\n  File \\\"/DATA/chowdari1/.local/lib/python3.7/site-packages/ray/tune/trial_runner.py\\\", line 924, in _process_trial\\n    results = self.trial_executor.fetch_result(trial)\\n  File \\\"/DATA/chowdari1/.local/lib/python3.7/site-packages/ray/tune/ray_trial_executor.py\\\", line 787, in fetch_result\\n    result = ray.get(trial_future[0], timeout=DEFAULT_GET_TIMEOUT)\\n  File \\\"/DATA/chowdari1/.local/lib/python3.7/site-packages/ray/_private/client_mode_hook.py\\\", line 105, in wrapper\\n    return func(*args, **kwargs)\\n  File \\\"/DATA/chowdari1/.local/lib/python3.7/site-packages/ray/worker.py\\\", line 1713, in get\\n    raise value.as_instanceof_cause()\\nray.exceptions.RayTaskError(TuneError): \\u001b[36mray::ImplicitFunc.train_buffered()\\u001b[39m (pid=19707, ip=172.25.0.208, repr=<types.ImplicitFunc object at 0x7fcf1e51bf98>)\\n  File \\\"/DATA/chowdari1/.local/lib/python3.7/site-packages/ray/tune/trainable.py\\\", line 255, in train_buffered\\n    result = self.train()\\n  File \\\"/DATA/chowdari1/.local/lib/python3.7/site-packages/ray/tune/trainable.py\\\", line 314, in train\\n    result = self.step()\\n  File \\\"/DATA/chowdari1/.local/lib/python3.7/site-packages/ray/tune/function_runner.py\\\", line 381, in step\\n    self._report_thread_runner_error(block=True)\\n  File \\\"/DATA/chowdari1/.local/lib/python3.7/site-packages/ray/tune/function_runner.py\\\", line 532, in _report_thread_runner_error\\n    (\\\"Trial raised an exception. Traceback:\\\\n{}\\\".format(err_tb_str)\\nray.tune.error.TuneError: Trial raised an exception. Traceback:\\n\\u001b[36mray::ImplicitFunc.train_buffered()\\u001b[39m (pid=19707, ip=172.25.0.208, repr=<types.ImplicitFunc object at 0x7fcf1e51bf98>)\\n  File \\\"/DATA/chowdari1/.local/lib/python3.7/site-packages/ray/tune/function_runner.py\\\", line 262, in run\\n    self._entrypoint()\\n  File \\\"/DATA/chowdari1/.local/lib/python3.7/site-packages/ray/tune/function_runner.py\\\", line 331, in entrypoint\\n    self._status_reporter.get_checkpoint())\\n  File \\\"/DATA/chowdari1/.local/lib/python3.7/site-packages/ray/tune/function_runner.py\\\", line 597, in _trainable_func\\n    output = fn()\\n  File \\\"/tmp/ipykernel_3885/2932301558.py\\\", line 4, in train\\n  File \\\"/tmp/ipykernel_3885/1736512985.py\\\", line 6, in __init__\\nNameError: name 'VariationalEncoder' is not defined\\n\",\n  \"trial_name_creator\": null,\n  \"custom_trial_name\": null,\n  \"custom_dirname\": null,\n  \"saving_to\": null,\n  \"remote_checkpoint_dir_prefix\": null,\n  \"sync_function_tpl\": null,\n  \"checkpoint_freq\": 0,\n  \"checkpoint_at_end\": false,\n  \"keep_checkpoints_num\": null,\n  \"checkpoint_score_attr\": \"training_iteration\",\n  \"sync_on_checkpoint\": true,\n  \"checkpoint_manager\": {\n    \"_type\": \"CLOUDPICKLE_FALLBACK\",\n    \"value\": \"80059566010000000000008c1b7261792e74756e652e636865636b706f696e745f6d616e61676572948c11436865636b706f696e744d616e616765729493942981947d94288c146b6565705f636865636b706f696e74735f6e756d94477ff00000000000008c165f636865636b706f696e745f73636f72655f6465736394898c165f636865636b706f696e745f73636f72655f61747472948c12747261696e696e675f697465726174696f6e948c1c6e65776573745f70657273697374656e745f636865636b706f696e749468008c0a436865636b706f696e749493942981947d94288c0773746f72616765948c0a70657273697374656e74948c0576616c7565944e8c06726573756c74947d9475628c195f6e65776573745f6d656d6f72795f636865636b706f696e7494680b2981947d9428680e8c066d656d6f72799468104e68117d9475628c115f626573745f636865636b706f696e7473945d948c0b5f6d656d62657273686970948f9475622e\"\n  },\n  \"restore_path\": null,\n  \"restoring_from\": null,\n  \"num_failures\": 1,\n  \"has_new_resources\": false,\n  \"results\": \"80054e2e\",\n  \"best_result\": \"80054e2e\",\n  \"param_config\": \"80054e2e\",\n  \"extra_arg\": \"80054e2e\",\n  \"_state_json\": null,\n  \"_state_valid\": false\n}",
    "{\n  \"stub\": false,\n  \"trainable_name\": \"DEFAULT\",\n  \"trial_id\": \"5aaa5_00001\",\n  \"config\": {\n    \"l1\": 224,\n    \"l2\": 20,\n    \"lr\": 0.0008553004842792813,\n    \"batch_size\": 256\n  },\n  \"local_dir\": \"/DATA/chowdari1/ray_results/DEFAULT_2022-02-09_01-34-42\",\n  \"evaluated_params\": {\n    \"l1\": 224,\n    \"l2\": 20,\n    \"lr\": 0.0008553004842792813,\n    \"batch_size\": 256\n  },\n  \"experiment_tag\": \"1_batch_size=256,l1=224,l2=20,lr=0.0008553\",\n  \"location\": {\n    \"_type\": \"CLOUDPICKLE_FALLBACK\",\n    \"value\": \"8005953a000000000000008c0e7261792e74756e652e747269616c948c084c6f636174696f6e9493942981947d94288c08686f73746e616d65944e8c03706964944e75622e\"\n  },\n  \"placement_group_factory\": \"800595b6000000000000008c1f7261792e74756e652e7574696c732e706c6163656d656e745f67726f757073948c15506c6163656d656e7447726f7570466163746f72799493942981947d94288c155f686561645f62756e646c655f69735f656d70747994898c085f62756e646c6573945d947d94288c03435055944740000000000000008c0347505594473fe000000000000075618c095f7374726174656779948c045041434b948c055f6172677394298c075f6b7761726773947d9475622e\",\n  \"stopping_criterion\": {},\n  \"log_to_file\": [\n    null,\n    null\n  ],\n  \"max_failures\": 0,\n  \"_last_result\": {\n    \"trial_id\": \"5aaa5_00001\"\n  },\n  \"_default_result_or_future\": null,\n  \"last_update_time\": -Infinity,\n  \"metric_analysis\": {},\n  \"n_steps\": [\n    5,\n    10\n  ],\n  \"metric_n_steps\": {},\n  \"export_formats\": [],\n  \"status\": \"ERROR\",\n  \"start_time\": 1644350685.059893,\n  \"logdir\": \"/DATA/chowdari1/ray_results/DEFAULT_2022-02-09_01-34-42/DEFAULT_5aaa5_00001_1_batch_size=256,l1=224,l2=20,lr=0.0008553_2022-02-09_01-34-45\",\n  \"runner\": null,\n  \"last_debug\": 0,\n  \"error_file\": \"/DATA/chowdari1/ray_results/DEFAULT_2022-02-09_01-34-42/DEFAULT_5aaa5_00001_1_batch_size=256,l1=224,l2=20,lr=0.0008553_2022-02-09_01-34-45/error.txt\",\n  \"error_msg\": \"Traceback (most recent call last):\\n  File \\\"/DATA/chowdari1/.local/lib/python3.7/site-packages/ray/tune/trial_runner.py\\\", line 924, in _process_trial\\n    results = self.trial_executor.fetch_result(trial)\\n  File \\\"/DATA/chowdari1/.local/lib/python3.7/site-packages/ray/tune/ray_trial_executor.py\\\", line 787, in fetch_result\\n    result = ray.get(trial_future[0], timeout=DEFAULT_GET_TIMEOUT)\\n  File \\\"/DATA/chowdari1/.local/lib/python3.7/site-packages/ray/_private/client_mode_hook.py\\\", line 105, in wrapper\\n    return func(*args, **kwargs)\\n  File \\\"/DATA/chowdari1/.local/lib/python3.7/site-packages/ray/worker.py\\\", line 1713, in get\\n    raise value.as_instanceof_cause()\\nray.exceptions.RayTaskError(TuneError): \\u001b[36mray::ImplicitFunc.train_buffered()\\u001b[39m (pid=19755, ip=172.25.0.208, repr=<types.ImplicitFunc object at 0x7f46150f7668>)\\n  File \\\"/DATA/chowdari1/.local/lib/python3.7/site-packages/ray/tune/trainable.py\\\", line 255, in train_buffered\\n    result = self.train()\\n  File \\\"/DATA/chowdari1/.local/lib/python3.7/site-packages/ray/tune/trainable.py\\\", line 314, in train\\n    result = self.step()\\n  File \\\"/DATA/chowdari1/.local/lib/python3.7/site-packages/ray/tune/function_runner.py\\\", line 381, in step\\n    self._report_thread_runner_error(block=True)\\n  File \\\"/DATA/chowdari1/.local/lib/python3.7/site-packages/ray/tune/function_runner.py\\\", line 532, in _report_thread_runner_error\\n    (\\\"Trial raised an exception. Traceback:\\\\n{}\\\".format(err_tb_str)\\nray.tune.error.TuneError: Trial raised an exception. Traceback:\\n\\u001b[36mray::ImplicitFunc.train_buffered()\\u001b[39m (pid=19755, ip=172.25.0.208, repr=<types.ImplicitFunc object at 0x7f46150f7668>)\\n  File \\\"/DATA/chowdari1/.local/lib/python3.7/site-packages/ray/tune/function_runner.py\\\", line 262, in run\\n    self._entrypoint()\\n  File \\\"/DATA/chowdari1/.local/lib/python3.7/site-packages/ray/tune/function_runner.py\\\", line 331, in entrypoint\\n    self._status_reporter.get_checkpoint())\\n  File \\\"/DATA/chowdari1/.local/lib/python3.7/site-packages/ray/tune/function_runner.py\\\", line 597, in _trainable_func\\n    output = fn()\\n  File \\\"/tmp/ipykernel_3885/2932301558.py\\\", line 4, in train\\n  File \\\"/tmp/ipykernel_3885/1736512985.py\\\", line 6, in __init__\\nNameError: name 'VariationalEncoder' is not defined\\n\",\n  \"trial_name_creator\": null,\n  \"custom_trial_name\": null,\n  \"custom_dirname\": null,\n  \"saving_to\": null,\n  \"remote_checkpoint_dir_prefix\": null,\n  \"sync_function_tpl\": null,\n  \"checkpoint_freq\": 0,\n  \"checkpoint_at_end\": false,\n  \"keep_checkpoints_num\": null,\n  \"checkpoint_score_attr\": \"training_iteration\",\n  \"sync_on_checkpoint\": true,\n  \"checkpoint_manager\": {\n    \"_type\": \"CLOUDPICKLE_FALLBACK\",\n    \"value\": \"80059566010000000000008c1b7261792e74756e652e636865636b706f696e745f6d616e61676572948c11436865636b706f696e744d616e616765729493942981947d94288c146b6565705f636865636b706f696e74735f6e756d94477ff00000000000008c165f636865636b706f696e745f73636f72655f6465736394898c165f636865636b706f696e745f73636f72655f61747472948c12747261696e696e675f697465726174696f6e948c1c6e65776573745f70657273697374656e745f636865636b706f696e749468008c0a436865636b706f696e749493942981947d94288c0773746f72616765948c0a70657273697374656e74948c0576616c7565944e8c06726573756c74947d9475628c195f6e65776573745f6d656d6f72795f636865636b706f696e7494680b2981947d9428680e8c066d656d6f72799468104e68117d9475628c115f626573745f636865636b706f696e7473945d948c0b5f6d656d62657273686970948f9475622e\"\n  },\n  \"restore_path\": null,\n  \"restoring_from\": null,\n  \"num_failures\": 1,\n  \"has_new_resources\": false,\n  \"results\": \"80054e2e\",\n  \"best_result\": \"80054e2e\",\n  \"param_config\": \"80054e2e\",\n  \"extra_arg\": \"80054e2e\",\n  \"_state_json\": null,\n  \"_state_valid\": false\n}",
    "{\n  \"stub\": false,\n  \"trainable_name\": \"DEFAULT\",\n  \"trial_id\": \"5aaa5_00005\",\n  \"config\": {\n    \"l1\": 224,\n    \"l2\": 20,\n    \"lr\": 0.00013172596006462482,\n    \"batch_size\": 256\n  },\n  \"local_dir\": \"/DATA/chowdari1/ray_results/DEFAULT_2022-02-09_01-34-42\",\n  \"evaluated_params\": {\n    \"l1\": 224,\n    \"l2\": 20,\n    \"lr\": 0.00013172596006462482,\n    \"batch_size\": 256\n  },\n  \"experiment_tag\": \"5_batch_size=256,l1=224,l2=20,lr=0.00013173\",\n  \"location\": {\n    \"_type\": \"CLOUDPICKLE_FALLBACK\",\n    \"value\": \"8005953a000000000000008c0e7261792e74756e652e747269616c948c084c6f636174696f6e9493942981947d94288c08686f73746e616d65944e8c03706964944e75622e\"\n  },\n  \"placement_group_factory\": \"800595b6000000000000008c1f7261792e74756e652e7574696c732e706c6163656d656e745f67726f757073948c15506c6163656d656e7447726f7570466163746f72799493942981947d94288c155f686561645f62756e646c655f69735f656d70747994898c085f62756e646c6573945d947d94288c03435055944740000000000000008c0347505594473fe000000000000075618c095f7374726174656779948c045041434b948c055f6172677394298c075f6b7761726773947d9475622e\",\n  \"stopping_criterion\": {},\n  \"log_to_file\": [\n    null,\n    null\n  ],\n  \"max_failures\": 0,\n  \"_last_result\": {\n    \"trial_id\": \"5aaa5_00005\"\n  },\n  \"_default_result_or_future\": null,\n  \"last_update_time\": -Infinity,\n  \"metric_analysis\": {},\n  \"n_steps\": [\n    5,\n    10\n  ],\n  \"metric_n_steps\": {},\n  \"export_formats\": [],\n  \"status\": \"ERROR\",\n  \"start_time\": 1644350685.1927958,\n  \"logdir\": \"/DATA/chowdari1/ray_results/DEFAULT_2022-02-09_01-34-42/DEFAULT_5aaa5_00005_5_batch_size=256,l1=224,l2=20,lr=0.00013173_2022-02-09_01-34-45\",\n  \"runner\": null,\n  \"last_debug\": 0,\n  \"error_file\": \"/DATA/chowdari1/ray_results/DEFAULT_2022-02-09_01-34-42/DEFAULT_5aaa5_00005_5_batch_size=256,l1=224,l2=20,lr=0.00013173_2022-02-09_01-34-45/error.txt\",\n  \"error_msg\": \"Traceback (most recent call last):\\n  File \\\"/DATA/chowdari1/.local/lib/python3.7/site-packages/ray/tune/trial_runner.py\\\", line 924, in _process_trial\\n    results = self.trial_executor.fetch_result(trial)\\n  File \\\"/DATA/chowdari1/.local/lib/python3.7/site-packages/ray/tune/ray_trial_executor.py\\\", line 787, in fetch_result\\n    result = ray.get(trial_future[0], timeout=DEFAULT_GET_TIMEOUT)\\n  File \\\"/DATA/chowdari1/.local/lib/python3.7/site-packages/ray/_private/client_mode_hook.py\\\", line 105, in wrapper\\n    return func(*args, **kwargs)\\n  File \\\"/DATA/chowdari1/.local/lib/python3.7/site-packages/ray/worker.py\\\", line 1713, in get\\n    raise value.as_instanceof_cause()\\nray.exceptions.RayTaskError(TuneError): \\u001b[36mray::ImplicitFunc.train_buffered()\\u001b[39m (pid=19776, ip=172.25.0.208, repr=<types.ImplicitFunc object at 0x7fbe3f5a5b38>)\\n  File \\\"/DATA/chowdari1/.local/lib/python3.7/site-packages/ray/tune/trainable.py\\\", line 255, in train_buffered\\n    result = self.train()\\n  File \\\"/DATA/chowdari1/.local/lib/python3.7/site-packages/ray/tune/trainable.py\\\", line 314, in train\\n    result = self.step()\\n  File \\\"/DATA/chowdari1/.local/lib/python3.7/site-packages/ray/tune/function_runner.py\\\", line 381, in step\\n    self._report_thread_runner_error(block=True)\\n  File \\\"/DATA/chowdari1/.local/lib/python3.7/site-packages/ray/tune/function_runner.py\\\", line 532, in _report_thread_runner_error\\n    (\\\"Trial raised an exception. Traceback:\\\\n{}\\\".format(err_tb_str)\\nray.tune.error.TuneError: Trial raised an exception. Traceback:\\n\\u001b[36mray::ImplicitFunc.train_buffered()\\u001b[39m (pid=19776, ip=172.25.0.208, repr=<types.ImplicitFunc object at 0x7fbe3f5a5b38>)\\n  File \\\"/DATA/chowdari1/.local/lib/python3.7/site-packages/ray/tune/function_runner.py\\\", line 262, in run\\n    self._entrypoint()\\n  File \\\"/DATA/chowdari1/.local/lib/python3.7/site-packages/ray/tune/function_runner.py\\\", line 331, in entrypoint\\n    self._status_reporter.get_checkpoint())\\n  File \\\"/DATA/chowdari1/.local/lib/python3.7/site-packages/ray/tune/function_runner.py\\\", line 597, in _trainable_func\\n    output = fn()\\n  File \\\"/tmp/ipykernel_3885/2932301558.py\\\", line 4, in train\\n  File \\\"/tmp/ipykernel_3885/1736512985.py\\\", line 6, in __init__\\nNameError: name 'VariationalEncoder' is not defined\\n\",\n  \"trial_name_creator\": null,\n  \"custom_trial_name\": null,\n  \"custom_dirname\": null,\n  \"saving_to\": null,\n  \"remote_checkpoint_dir_prefix\": null,\n  \"sync_function_tpl\": null,\n  \"checkpoint_freq\": 0,\n  \"checkpoint_at_end\": false,\n  \"keep_checkpoints_num\": null,\n  \"checkpoint_score_attr\": \"training_iteration\",\n  \"sync_on_checkpoint\": true,\n  \"checkpoint_manager\": {\n    \"_type\": \"CLOUDPICKLE_FALLBACK\",\n    \"value\": \"80059566010000000000008c1b7261792e74756e652e636865636b706f696e745f6d616e61676572948c11436865636b706f696e744d616e616765729493942981947d94288c146b6565705f636865636b706f696e74735f6e756d94477ff00000000000008c165f636865636b706f696e745f73636f72655f6465736394898c165f636865636b706f696e745f73636f72655f61747472948c12747261696e696e675f697465726174696f6e948c1c6e65776573745f70657273697374656e745f636865636b706f696e749468008c0a436865636b706f696e749493942981947d94288c0773746f72616765948c0a70657273697374656e74948c0576616c7565944e8c06726573756c74947d9475628c195f6e65776573745f6d656d6f72795f636865636b706f696e7494680b2981947d9428680e8c066d656d6f72799468104e68117d9475628c115f626573745f636865636b706f696e7473945d948c0b5f6d656d62657273686970948f9475622e\"\n  },\n  \"restore_path\": null,\n  \"restoring_from\": null,\n  \"num_failures\": 1,\n  \"has_new_resources\": false,\n  \"results\": \"80054e2e\",\n  \"best_result\": \"80054e2e\",\n  \"param_config\": \"80054e2e\",\n  \"extra_arg\": \"80054e2e\",\n  \"_state_json\": null,\n  \"_state_valid\": false\n}"
  ],
  "runner_data": {
    "_max_pending_trials": 52,
    "_metric": null,
    "_total_time": 0,
    "_iteration": 21,
    "_has_errored": true,
    "_fail_fast": false,
    "_server_port": null,
    "_cached_trial_decisions": {},
    "_queued_trial_decisions": {},
    "_updated_queue": true,
    "_result_wait_time": 1,
    "_should_stop_experiment": false,
    "_local_checkpoint_dir": "/DATA/chowdari1/ray_results/DEFAULT_2022-02-09_01-34-42",
    "_remote_checkpoint_dir": null,
    "_stopper": {
      "_type": "CLOUDPICKLE_FALLBACK",
      "value": "80059527000000000000008c107261792e74756e652e73746f70706572948c0b4e6f6f7053746f707065729493942981942e"
    },
    "_resumed": false,
    "_start_time": 1644350682.624544,
    "_last_checkpoint_time": -Infinity,
    "_session_str": "2022-02-09_01-34-42",
    "checkpoint_file": "/DATA/chowdari1/ray_results/DEFAULT_2022-02-09_01-34-42/experiment_state-2022-02-09_01-34-42.json",
    "_checkpoint_period": "auto",
    "launch_web_server": false
  },
  "stats": {
    "start_time": 1644350682.624544,
    "timestamp": 1644350683.07615
  }
}